api/src/main/java/com/exelaration/abstractmemery/services/implementations/FileStorageServiceImpl.java

```diff
@@ -22,7 +22,7 @@ public void save(String fileName, String urlData, String storageLocation) {
     try {
       fileLocation = (uploadingDir + fileName);
       Path path = Paths.get(fileLocation);
-      byte[] decodedBytes = Base64.getDecoder().decode(urlData);
+      byte[] decodedBytes = Base64.decodeBase64(urlData);
```

Comment:
Why was this changed to use the Tomcat decoder?

Reply:
idk, what dependency was it using before